components/Footer.tsx

```diff
@@ -42,25 +42,25 @@ const Footer = () => {
                 </div>
               ))}
             </div>
+        </div>
 
-            <div className="flex justify-between
-            items-center flex-wrap mt-10 border-t
-            border-gray-100 sm:px-16 px-6 py-10">
-              <p>
-                @2024 CarHub. All Rights Reserved
-              </p>
+        <div className="flex justify-between

```

Comment:
It would be better:
separating between rematching the div tag and modifying these indentations.